docs/source/concepts/processes.rst

```diff
@@ -163,3 +163,20 @@ A process checkpoint is a complete representation of a ``Process`` instance in m
 Since it is a complete representation, the ``Process`` instance can also be fully reconstructed from such a checkpoint.
 At any state transition of a process, a checkpoint will be created, by serializing the process instance and storing it as an attribute on the corresponding process node.
 This mechanism is the final cog in the machine, together with the persisted process queue of RabbitMQ as explained in the previous section, that allows processes to continue after the machine they were running on, has been shutdown and restarted.
+
+
+.. _concepts_process_sealing:
+
+Process sealing
+===============
+One of the cardinal rules of AiiDA is that once a node is *stored*, it is immutable, which means that its attributes can no longer be changed.
+This rule is a problem for processes, however, since in order to be able to start running it, its corresponding process node first has to be stored.
+However, at that point its attributes, such as the process state or other mutable attributes, can no longer be changed by the engine throughout the lifetime of the corresponding process.
+To overcome this limitation, the concept of *updatable* attributes are introduced.
```

Comment:
Either 
- the concept of *updatable* attributes is introduced

or
- *updatable* attributes are introduced

Reply:
Done